POUF1.md

```diff
@@ -0,0 +1,442 @@
+
+This POUF supports asymmetric keys and uses the [same keys for encrypting and signing](https://uptane.github.io/uptane-standard/uptane-standard.html#build-time-prerequisite-requirements-for-ecus). Using the same keys allows for a simplified reference implementation and less key management. In particular, each ECU has a private key and registers the associated public key with the director repository. The ECU will use the private key to sign messages and the director repository may use the public key to encrypt images per ECU. In addition, each metadata role has either online or offline private keys that are used on the director and image repositories. The public keys associated with these roles are used by the ECUs during validation.
+
+The Image repository and Director repository have public interfaces for communication with ECUs. To make the public interface, the Image and Directory repositories each store metadata in a filesystem that is hosted to form the public interface. These interfaces do not require authentication to read.
+
+The Director repository identifies a vehicle at the start of the update process using a stored vehicle version manifest. The Director checks the vehicle version manifest as described in the standard, but does not make any additional checks. This POUF does not do dependency resolution as the reference implementation is not used in practice and so does not have conflicts.
+
+This POUF does not support sending vehicle version manifests as [diffs](https://uptane.github.io/uptane-standard/uptane-standard.html#construct_manifest_primary) as the reference implementation is not concerned about potential bandwidth issues.
+
+# Metadata
+This POUF supports some metadata features that are optional in the Uptane Standard, but are required for an implementation of this POUF. These features are included in the metadata definitions in {Formats}. The details of these features are:
+* The metadata version will increment on an update to the metadata.
+* Root metadata must support [mappings of roles to lists of URLs as described in TAP 5](https://uptane.github.io/uptane-standard/uptane-standard.html#root_meta).
+* Repository mapping metadata will be included using a map file. The format of this file is described in {Formats}. In the mapping metadata, files are expressed using an enumerated list that supports wildcards.
+* The OEM will provide unencrypted images to ECUs, hence the Director repository’s targets metadata will contain hashes of unencrypted images. Metadata about encrypted images may be included in the custom field of targets metadata as described in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images).
```

Comment:
I thought encryption was optionally supported as was discussed above...

Reply:
This is referring to images that are encrypted per ECU. The Image repository does not know which ECU will receive the image, and so only contains metadata for the unencrypted image. The Director knows the ECU that will receive the image and so can add the encrypted metadata to the custom field.

I tried to make this more clear in the description.